include/ddc/traits.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <type_traits>
+
+#include "non_uniform_discretization.hpp"
+#include "uniform_discretization.hpp"
+
+template <class T>
+struct is_uniform_domain : std::false_type
+{
+};
+
+template <class... DDims>
+struct is_uniform_domain<DiscreteDomain<DDims...>>
```

Comment:
shouldn't you include something for `DiscreteDomain`?

Reply:
I think a declaration is missing in this case.

Reply:
Not sure to understand

Reply:
I mean this:
```suggestion

template <class... DDims>
class DiscreteDomain<DDims...>;

struct is_uniform_domain<DiscreteDomain<DDims...>>
```